scripts/quests/windurst/Reap_What_You_Sow.lua

```diff
@@ -33,28 +33,31 @@ quest.sections =
 
         [xi.zone.WINDURST_WATERS] =
         {
-            ['Mashuu-Ajuu'] =
-            {
-                onTrigger = function(player, npc)
-                    if player:getFameLevel(xi.quest.fame_area.WINDURST) >= 4 then
-                        return quest:progressEvent(483)
-                    else
-                        return quest:progressEvent(463, 0, xi.items.SOBBING_FUNGUS, xi.items.BAG_OF_HERB_SEEDS)
-                    end
-                end,
-            },
+            -- Unsure where event(483) should be...
+            ['Mashuu-Ajuu'] = quest:progressEvent(463, 0, xi.items.SOBBING_FUNGUS, xi.items.BAG_OF_HERB_SEEDS),
```

Comment:
Sorry I meant for that if logic to go here. You can remove the logic in the quest_available section above and put it down below on Mashuu-Ajuu's trigger.

if
    fame >= 4 and
    not player:hasCompletedQuest(xi.quest.log_id.WINDURST, xi.quest.id.windurst.LET_SLEEPING_DOGS_LIE))
then
    event 483
else
    progressEvent 463
    
If that makes sense

Reply:
![image](https://github.com/AirSkyBoat/AirSkyBoat/assets/62350957/d7126da7-fecc-4e1f-bf79-411c90e199f0)

Something like this

Reply:
my bad haha that does make more sense